### PR TITLE
Disable exponential backoff for development

### PR DESCRIPTION
**Pull request by byucesoy**
It is likely to have errors/exceptions during development. Exponential backoff slows us down while doing fast iterative development. Thus, I disabled it for development environments. Still, we put 5 seconds delay because if we schedule next run to now(), it gets executed almost immediatelly and causing lots of log generation, which also degrades the development experience.

### CODE REVIEW

File: model/strand.rb [hunk 1]
```diff
@@ -36,9 +36,10 @@ def self.prog_verify(prog)
   end
 
   def self.lease(id)
+    schedule = Config.development? ? "(now() + least(5, try) * '1 second'::interval)" : "(now() + least(2 ^ least(try, 20), 600) * random() * '1 second'::interval)"
```
Comment (fdr) [inline model/strand.rb:39]: How about putting this into a const, so it's computed exactly once at start-up?

Comment (byucesoy) [reply model/strand.rb]: Makes sense. I did that. Funnily, it made testing significantly difficult as that line would be executed at load time. So I added :nocov:. Any suggestions to test it?

Comment (enescakir) [reply model/strand.rb]: We have encountered the same issue in several places. The only solution I found for reloading constant values was by using ﻿`load "FILEPATH"`. Consequently, we decided to add ﻿`:nocov:` to them.

Comment (fdr) [reply model/strand.rb]: I think nocov is fine for this case. Automation of coverage inspection is more important than the branch coverage per se.

**Review by fdr — APPROVED** [review]

**Pull request merged**